src/page-multi-facility/ScenarioSidebar.tsx

```diff
@@ -51,7 +52,9 @@ const ScenarioSidebar: React.FC<Props> = (props) => {
 
   const handleScenarioChange = (scenarioChange: object) => {
     const changes = Object.assign({}, scenario, scenarioChange);
-    updateScenario(changes);
+    saveScenario(changes).then((_) => {
```

Comment:
Why do we need to first `save` then `update` here?

Reply:
Yeah, that's pretty confusing.  I'll update to `dispatchScenarioUpdate`